ethers-core/src/types/transaction/eip2718.rs

```diff
@@ -491,6 +574,11 @@ impl From<&Transaction> for TypedTransaction {
             // Optimism Deposited (0x7E)
             Some(x) if x == U64::from(0x7E) => {
                 let request: OptimismDepositedTransactionRequest = tx.into();
+            }
+            #[cfg(feature = "kroma")]
+            // Kroma Deposited transaction (0x7E)
```

Comment:
Let's match with optimism comment

```suggestion
            // Kroma Deposited (0x7E)
```